### ISSUE TITLE
[Bug] RemoveTag Responder Operation

### ISSUE THREAD

**Issue by crackytsi** [TheHive-Project/TheHive#1393]
### Request Type
Feature Request

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | Debian
| OS version (client)       | 10
| TheHive version / git hash   | 3.4
| Package Type              | DEB

### Problem Description
I want to remove a Case-Tag after unblocking an IP on the Firewall (by a responder).
Unforunately TheHive currently supports only the following responder Operations:
```
    case a: AddTagToCase      ⇒ addTagToCaseWrites.writes(a)
    case a: AddTagToArtifact  ⇒ addTagToArtifactWrites.writes(a)
    case a: CreateTask        ⇒ createTaskWrites.writes(a)
    case a: AddCustomFields   ⇒ addCustomFieldsWrites.writes(a)
    case a: CloseTask         ⇒ closeTaskWrites.writes(a)
    case a: MarkAlertAsRead   ⇒ markAlertAsReadWrites.writes(a)
    case a: AddLogToTask      ⇒ addLogToTaskWrites.writes(a)
    case a: AddTagToAlert     ⇒ addTagToAlertWrites.writes(a)
    case a: AddArtifactToCase ⇒ addArtifactToCaseWrites.writes(a)
    case a: AssignCase        ⇒ assignCaseWrites.writes(a)
```

Please support also the features:
-removeTagFromCase
-removeTagFromArtifact
-removeTagFromAlert
-MarkAlertAsUnread (I currently haven't UC in my mind, but I would't say that this can't happen)

Of course it would be nice to have that for TH4 as well ;)
Thanks :)

**Comment by crackytsi** [comment]
@To-om : Is this something you can do in one of the next milestones? I guess it should't to much work, but it would help me a lot...
